templates/README.md.mustache

```diff
@@ -154,25 +155,35 @@ local red = require('tinted-colorscheme').colors.base08
 value, "tinted-nvim-default" (`base16-ayu-dark`) will be used as a fallback
 theme.
 
-To have tinted-nvim update the theme when `tinty apply ayu-dark` is run, you
-need to tell Neovim to update and the easiest way to do this is to update the
-theme when Neovim is given focus. The following will update the theme, on window
-focus, if the theme has changed:
+To have tinted-nvim automatically update the theme when `tinty apply ayu-dark` is run, you can enable
+the live-reload feature.
 
 ```lua
 local tinted = require('tinted-colorscheme')
-tinted.setup()
-
-vim.api.nvim_create_autocmd("FocusGained", {
-  callback = function() tinted.setup() end
+tinted.setup(nil, {
+  supports = {
+    live_reload = true
+  }
 })
 ```
 
-**Note**: If you don't see colours, try adding `vim.opt.termguicolors = true` to
+The live-reload feature depends on [fwatch.nvim]. Add this dependency in whatever manner your plugin manager allows you.
+For example, with **lazy.nvim**:
+
+```diff
+ return {
+   "tinted-theming/tinted-nvim",
++  dependencies = {
++    { "rktjmp/fwatch.nvim" },
++  }
+ }
+```
+
+> ![NOTE]
```

Comment:
Is this needed? Not sure if it was a note for yourself or if it affects the markdown somehow.

Reply:
This is a Github-flavored Markdown thing:

> [!NOTE]
> This is w/ `!NOTE`

> [!TIP]
> This is w/ `!TIP`

> [!WARNING]
> This is w/ `!WARNING`

> [!CAUTION]
> This is w/ `!CAUTION`

Reply:
Ah ok nice didn’t know about it